catalyst_voices/apps/voices/e2e_tests/Earthfile

```diff
@@ -0,0 +1,49 @@
+VERSION 0.8
+
+# Base setup for E2E test runner
+# Uses official Playwright image which includes Chromium, Firefox, and WebKit
+builder:
+    FROM mcr.microsoft.com/playwright:v1.57.0-noble
+    WORKDIR /workspace
+
+    # Install additional dependencies for headed mode with Xvfb
+    RUN apt-get update && apt-get install -y \
+        xvfb \
+        && apt-get clean \
+        && rm -rf /var/lib/apt/lists/*
+
+    # Copy test files
+    COPY package.json package-lock.json ./
+    RUN npm ci
+
+    # Copy all test-related files
+    COPY . .
+
+# Build the E2E test runner image
+docker:
+    FROM +builder
+
+    ARG container="e2e-test-runner"
+    ARG tag="latest"
+    
+    # Set working directory
+    WORKDIR /workspace
+
+    # Default command to run tests
+    CMD ["npx", "playwright", "test"]
```

Comment:
You are setting the cmd here and then overriding it in the docker-compose file. Can you just put the `command:
      [
        "xvfb-run",
        "--auto-servernum",
        "--server-args=-screen 0 1920x1080x24",
        "npx",
        "playwright",
        "test",
        "--project=local-run",
      ]` here?